frontend/views/Config.tsx

```diff
@@ -20,7 +24,16 @@ interface GameListProps{
 const GameList: React.FC<GameListProps> = (props: GameListProps) => <React.Fragment>
 	{props.games.map(game=> <GamesCheck key={game.name} game={game.name}/>)};
 </React.Fragment>;
-export class Config extends React.Component {
+
+interface ConfigProps{
+	store: Store<AppStore, AppActions>;
+}
+
+export class Config extends React.Component<ConfigProps> {
```

Comment:
Rename this class to `ConfigView` maybe. At the moment the name suggests that it's configuration details rather than a view. Do the same for `Home` for consistency!